README.md

```diff
@@ -1 +1,7 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+This is a webpage built for microverse.
+
+## About
+
+webpage contains only a [header].
```

Comment:
- [ ] Kindly add more content to your readme file, ensure to use [this template ](https://github.com/microverseinc/readme-template)